### PR TITLE
Follow up updates to v1.24 release blog regarding CSI Snapshot

### PR DESCRIPTION
**Pull request by Debanitrkl**
Updated the Kubernetes v1.24 release blog with a missing major theme regarding CSI Snapshot.
cc @mickeyboxell @sftim

### CODE REVIEW

File: content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 1]
```diff
@@ -118,6 +118,11 @@ With containerd v1.6.0–v1.6.3, if you do not upgrade the CNI plugins and/or
 declare the CNI config version, you might encounter the following "Incompatible
 CNI versions" or "Failed to destroy network for sandbox" error conditions.
 
+## CSI Snapshot
+
+[VolumeSnapshot v1beta1 CRD has been removed](https://github.com/kubernetes/enhancements/issues/177). 
+Volume snapshot and restore functionality for Kubernetes and the Container Storage Interface (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, entered beta in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.21 and is now unsupported. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and [kubernetes-csi/external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.1.0) for more information.
```
Comment (xing-yang) [inline content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md:124]: VolumeSnapshot API moved to GA in v1.20 so v1beta1 API was deprecated in v1.20. In v1.21, we changed the Storage version from v1beta1 to v1.
I think we can refer to the KEP and the GA blog https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/.  We don't need to refer to this particular release tag: https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v4.1.0

**Review by sftim — COMMENTED** [review]
LGTM
for blog team

File: content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md [hunk 2]
```diff
@@ -118,6 +118,11 @@ With containerd v1.6.0–v1.6.3, if you do not upgrade the CNI plugins and/or
 declare the CNI config version, you might encounter the following "Incompatible
 CNI versions" or "Failed to destroy network for sandbox" error conditions.
 
+## CSI Snapshot
+
```
Comment (sftim) [inline content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md:122]:
```suggestion

_This information was added after initial publication._

```

File: content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md (hunk 1 above)
Comment (xing-yang) [reply content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md:124]: Suggest change to the following:
Volume snapshot and restore functionality for Kubernetes and the Container Storage Interface (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, moved to GA in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.20 and is now unsupported. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and [Volume Snapshot GA blog](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/) for more information.

Comment (xing-yang) [inline content/en/blog/_posts/2022-05-03-kubernetes-release-1.24.md:124]:
```suggestion
Volume snapshot and restore functionality for Kubernetes and the Container Storage Interface (CSI), which provides standardized APIs design (CRDs) and adds PV snapshot/restore support for CSI volume drivers, moved to GA in v1.20. VolumeSnapshot v1beta1 was deprecated in v1.20 and is now unsupported. Refer to [KEP-177: CSI Snapshot](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/177-volume-snapshot#kep-177-csi-snapshot) and [Volume Snapshot GA blog](https://kubernetes.io/blog/2020/12/10/kubernetes-1.20-volume-snapshot-moves-to-ga/) for more information.
```

**Review by sftim — COMMENTED** [review]
/approve

**Pull request merged**